pkg/controller/cassandradatacenter/configmap.go

```diff
@@ -60,14 +58,50 @@ func createOrUpdateOperatorConfigMap(rctx *reconciliationRequestContext, seedNod
 	return configVolume, nil
 }
 
-func addCassandraGossipingPropertyFileSnitchProperties(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, addFileFn func(path string, data string)) {
+func createOrUpdateCassandraRackConfig(rctx *reconciliationRequestContext, rack string) (*corev1.Volume, error) {
+
+	configMap := &corev1.ConfigMap{ObjectMeta: DataCenterResourceMetadata(rctx.cdc, "operator-rack-config")}
```

Comment:
Shouldn't the ConfigMap name contain the name of the rack? Otherwise this would overwrite when there is multiple racks, no?

Reply:
yes, definitely.